hydra-node/test/Hydra/Chain/Direct/Contract/Contest.hs

```diff
@@ -106,6 +106,7 @@ healthyContestSnapshot =
     , number = healthyContestSnapshotNumber
     , utxo = healthyContestUTxO
     , confirmed = []
+    , utxoToDecommit = Nothing
```

Comment:
why not arbitrary?

Reply:
We are not trying to test decommits here but in any case `utxoToDecommit` must be a subset of `utxo` from the snapshot so shouldn't have arbitrary value since ideally in tests we would decommit only what was committed before just like in real world (no on-chain code to enforce that yet)